tests/test_misc.py

```diff
@@ -2728,3 +2728,61 @@ def test_stkman_only(revault_network, bitcoind):
         rn.activate_vault(v)
     rn.unvault_vaults_anyhow(vaults)
     rn.cancel_vault(vaults[0])
+
+
+def test_getserverstatus(revault_network, bitcoind):
+    rn = revault_network
+    rn.deploy(n_stakeholders=2, n_managers=1)
+
+    # The coordinator is alive
+    for w in rn.participants():
+        res = w.rpc.call("getserverstatus")
+        assert res["coordinator"]["reachable"]
+        assert res["coordinator"]["host"] == f"127.0.0.1:{rn.coordinator_port}"
+
+    # The cosigners are alive, but only the managers see them
+    for w in rn.mans():
+        res = w.rpc.call("getserverstatus")
+        for cosigner in res["cosigners"]:
+            assert cosigner["reachable"]
+            # Sadly we don't persist the cosigner ports
+            assert cosigner["host"].startswith("127.0.0.1:")
```

Comment:
How comes?? How could you even `KKTransport::connect` without the port?

Reply:
Well I'm just checking if the host starts with `127.0.0.1`, then yes, there's a port after that... But I have no idea (from the test perspective) on what ports cosigners are running, so I can't check

Reply:
Yeah, brainfart, sorry.